### PR TITLE
Migration of carts from v1 to v2

### PR DESCRIPTION
**Pull request by jonmediajel**
https://github.com/MediaJel/mediajel-tracker/issues/381

### CODE REVIEW

File: src/v2/imports/carts/buddi.ts [hunk 1]
```diff
@@ -1,90 +1,57 @@
...
-        cartList.push(product);
-
-        window.tracker("trackAddToCart", {
-          sku: product.id.toString(),
-          name: product.name.toString() || "N/A",
-          category: "N/A",
-          unitPrice: parseFloat(product.price) || 0,
-          quantity: parseInt(product.qty) || 1,
-          currency: "USD",
+  buddiTrackerImport({
```
Comment (pacholoamit) [inline src/v2/imports/carts/buddi.ts:5]: Rename this to `buddiDataSource`

File: src/v2/imports/carts/yotpo.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          currency: item.currency,
+        });
+      });
+
+      window.tracker("trackTrans");
+    },
+  });
+};
+
+export default yotpoTracker;
```
Comment (pacholoamit) [inline src/v2/imports/carts/yotpo.ts:36]: You added a `yotpo` tracker but you did not add it to the index.ts in

```
mediajel-tracker/src/v2/imports/carts/index.ts
```

**Review by pacholoamit — CHANGES_REQUESTED** [review]
Please see changes

File: src/v2/imports/carts/yotpo.ts (hunk 2 above)
Comment (jonmediajel) [reply src/v2/imports/carts/yotpo.ts:36]: Done, Thanks for the feedback @pacholoamit

File: src/v2/imports/carts/buddi.ts (hunk 1 above)
Comment (jonmediajel) [reply src/v2/imports/carts/buddi.ts:5]: Done, Thanks for the feedback @pacholoamit

**Comment by jonmediajel** [comment]
> Please see changes

Thanks for the feedback @pacholoamit , feedbacks are now addressed

File: src/shared/utils/get-context.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ const getContext = (): QueryStringContext => {
 
   return {
     appId: appId || mediajelAppId, // Legacy support for old universal tag
-    version: version || "1",
+    version: version || "2", // tracker version
```
Comment (pacholoamit) [inline src/shared/utils/get-context.ts:15]: NO. don't do this

**Review by pacholoamit — CHANGES_REQUESTED** [review]
Please revert the version back

**Pull request merged**